packages/react-app/src/components/giveaways/cards/Card.tsx

```diff
@@ -68,14 +73,16 @@ const GiveawayCard = ({ giveaway, onWinnersGeneration }: GiveawayCardProps) => {
       }}
       elevation={0}
     >
-      {nrPendingParticipants !== undefined && nrPendingParticipants > 0 && (
-        <div className="card-pending-approvals">
-          <PendingApprovalBanner
-            giveaway={giveaway}
-            nrPending={nrPendingParticipants}
-          />
-        </div>
-      )}
+      {userInfo.role === UserRole.ADMIN &&
+        nrPendingParticipants !== undefined &&
+        nrPendingParticipants > 0 && (
```

Comment:
Since `nrPendingParticipants` is zero by default in the API and `nrPendingParticipants > 0` also filters the undefined 
```suggestion
      {userInfo.role === UserRole.ADMIN &&
        nrPendingParticipants > 0 && (
```

Reply:
I have to check if it is undefined, otherwise an error appears on condition `nrPendingParticipants > 0` `'nrPendingParticipants' is possibly 'undefined'`

Reply:
Isn't that and error on the type definition in the frontend since the backend never returns an undefined value for `nrPendingParticipants`?